video/out/opengl/ra.h

```diff
@@ -341,13 +331,18 @@ struct ra_fns {
 
     void (*buf_destroy)(struct ra *ra, struct ra_buf *buf);
 
-    // Essentially a fence: once the GPU uses the mapping for read-access (e.g.
-    // by starting a texture upload), the host must not write to the mapped
-    // data until an internal object has been signalled. This call returns
-    // whether it was signalled yet. If true, write accesses are allowed again.
-    // Optional, may be NULL if unavailable. This is only usable for buffers
-    // which have been persistently mapped.
-    bool (*poll_mapped_buffer)(struct ra *ra, struct ra_buf *buf);
+    // Update the contents of a buffer, starting at a given offset and up to a
+    // given size, with the contents of *data. This is an extremely common
+    // operation. Calling this while the buffer is considered "in use" is an
+    // error.
```

Comment:
Yes, it's also required for vertex buffers. But I think I changed my mind about vertex buffers, it's probably best to "hide" that inside ra_vk as well, so that's irrelevant. Also this technically applies to SSBOs as well, if you want to update them from the host it would have to be synchronized with the use by the shader.